4-WebApp-your-API/4-2-B2C/Client/appsettings.json

```diff
@@ -1,17 +1,23 @@
 ﻿{
-  "AzureAdB2C": {
-    "Instance": "https://fabrikamb2c.b2clogin.com",
-    "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
-    "Domain": "fabrikamb2c.onmicrosoft.com",
-    "SignedOutCallbackPath": "/signout/B2C_1_susi",
-    "SignUpSignInPolicyId": "b2c_1_susi",
-    "EditProfilePolicyId": "b2c_1_edit_profile" // Optional profile editing policy
-    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Warning"
-    }
-  },
-  "AllowedHosts": "*"
+    "AzureAdB2C": {
+        "Instance": "https://<your-b2c-name>.b2clogin.com",
+        "ClientId": "<b2c-webApp-clientId>",
+        "Domain": "<your-b2c-domain>",
```

Comment:
>Domain [](start = 9, length = 6)

this is the full tenant? like `fabrikamb2c.onmicrosoft.com` or a guid? can we use the word `Tenant` instead?